src/test/java/org/openstreetmap/atlas/generator/sharding/PbfVerifierTest.java

```diff
@@ -0,0 +1,79 @@
+package org.openstreetmap.atlas.generator.sharding;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.openstreetmap.atlas.geography.Rectangle;
+import org.openstreetmap.atlas.streaming.resource.InputStreamResource;
+import org.openstreetmap.atlas.streaming.resource.Resource;
+
+/**
+ * Tests functionality of the PbfVerifier.
+ *
+ * @author jamesgage
+ */
+public class PbfVerifierTest
+{
+    private final Resource slippyTileFile = new InputStreamResource(
+            () -> PbfVerifierTest.class.getResourceAsStream("testSlippyTile"));
+    private final Resource pbf1 = new InputStreamResource(
+            () -> PbfVerifierTest.class.getResourceAsStream("10-313-380.pbf"));
```

Comment:
What was the secret to getting this working?

Reply:
@matthieun suggested that the shardToBounds object was actually null, not the pbf resources themselves. He was right! Not sure how I missed that.

Reply:
Interesting. Was the slippy tile file parsing failing?